web/docs/installation/deploy-the-platform.md

```diff
@@ -4,39 +4,140 @@ sidebar_position: 5
 
 # Deploy the platform
 
-The *Tenzir Platform* manages nodes and the pipelines running on them, offering
-a visual interface to explore data, manage nodes, and pipelines, and create
-dashboards.
+The **Tenzir Platform** is the control plane that manage Tenzir Nodes. The
+platform also provides a web interface to explore data, create pipelines, and
+build dashboards.
 
 :::warning Sovereign Edition Required
-Tenzir offers a free and cloud-hosted version of the Tenzir Platform on
-[app.tenzir.com](https://app.tenzir.com) for all users of the Community Edition.
-This guide explains how to run the platform on your own premises as a user of
-the [Sovereign Edition](https://tenzir.com/pricing).
+Tenzir offers a free and cloud-hosted version of the Tenzir Platform at
+[app.tenzir.com](https://app.tenzir.com). This guide explains how to run the
+platform on your own premises using the [Sovereign
+Edition](https://tenzir.com/pricing).
 :::
 
-## Configuration
+## Download the Platform
 
-The platform requires some external infrastructure that must be installed and
-configured separately.
+Start with downloading the [latest Tenzir Platform
+release](https://github.com/tenzir/platform/releases/latest) and unpack the
+archive.
+
+## Set Up Docker Registry Access
+
+As part of your distribution, you were provided an authentication token
+(`YOUR_DOCKER_TOKEN` below) to fetch the Docker images. Log in with the token
+as follows:
+
+```bash
+echo YOUR_DOCKER_TOKEN | docker login ghcr.io -u tenzir-distribution --password-stdin
+```
+
+## Configure the Platform
+
+You need to configure a few external services to run the platform, such as a
+HTTP reverse proxy, an identity provider, and a state database.
+
+Scroll down to the [configuration options](#configuration-section) and populate
+a `.env` file with your individual settings.
```

Comment:
Link doesn't work.

Also, not sure whether it makes sense to put the configuration into its own section at the instead instead of just putting everything right here.

Reply:
The reason was to avoid getting into nitty gritty details here and keep the flow wrt. to the performed actions. This way it's a lot easier to see the steps at a glance. We tend to write emails to prospects this way, which seemed to be easier to grasp.